src/main.c

```diff
@@ -40,9 +40,23 @@ struct txn current_txn;
 static uint8_t msgpack_buf[1024];
 static unsigned int msgpack_next_off;
 
+/* The SDK exposes no information about the length of the received
```

Comment:
`io_exchange` appears to return a length, but it's unclear to me if an attacker can make it seem artifically large.